docs/UserGuide.adoc

```diff
@@ -183,197 +183,282 @@ Example:
 ==== Favoriting a person [Since v1.2]: `fav`
 
-Labels the specified person(s) as a favorite contact. +
-Command format: `fav INDEX [ADDITIONAL INDEXES]`
+To add person(s) as a favorite contact, use the `fav` command.
+
+====
+*Command format*: `[red]#fav# [green]#INDEX [ADDITIONAL INDEXES]...#`
+====
 
 ****
 * Adds the person(s) as a favorite contact at the specified `INDEX` or `INDEXES`.
 * The index refers to the index number shown in the most recently displayed list.
-* The index *must be a positive integer* 1, 2, 3, ...
+* The index must be a positive integer 1, 2, 3, ...
 ****
 
-Examples:
+*Examples*:
+
+* `[red]#fav# [green]#5 6#` +
+Adds the 5th and 6th person as favorite contacts in the most recently displayed list.
+
+.Added 5th and 6th persons as favorite contacts
+image::favorite-multiple-command-result.png[width="700"]
 
-* `fav 1 3` +
-Favorites the 1st and 3rd person in the most recently displayed list.
-* `find Betsy` +
-`unfav 1` +
-Favorites the first person in the list of contacts with the name `Betsy`.
+* `[red]#find# [green]#Bernice#` +
+`[red]#fav# [green]#1#` +
+Adds the first person in the list of contacts with the name `Bernice` as a favorite contact.
+
+.Added the first person i.e. Bernice as a favorite contact
+image::favorite-after-find-command-result.png[width="800"]
 
 ==== Unfavoriting a person [Since v1.2]: `unfav`
 
-Removes the the `favorite` label from the specified person(s). +
-Command format: `unfav INDEX [ADDITIONAL INDEXES]`
+To remove person(s) from the list of favorite contacts, use the `unfav` command.
+
+====
+*Command format*: `[red]#unfav# [green]#INDEX [ADDITIONAL INDEXES]...#`
+====
 
 ****
 * Removes the person(s) from the list of favorite contacts at the specified `INDEX` or `INDEXES`.
 * The index refers to the index number shown in the most recently displayed list.
-* The index *must be a positive integer* 1, 2, 3, ...
+* The index must be a positive integer 1, 2, 3, ...
 ****
 
-Examples:
+*Examples*:
+
+* `[red]#unfav# [green]#1 2#` +
+Removes the 1st and 2nd person from favorite contacts in the most recently displayed list.
 
-* `unfav 2 4` +
-Unfavorites the 2nd and 4th person in the most recently displayed list.
-* `find Betsy` +
-`unfav 1` +
-Unfavorites the first person in the list of contacts with the name `Betsy`.
+.Removed the 1st and 2nd person from favorite contacts
+image::unfavorite-multiple-command-result.png[width="700"]
 
+* `[red]#find# [green]#Alex#` +
+`[red]#unfav# [green]#1#` +
+Removes the first person in the list of contacts with the name `Alex`.
+
+.Removed person at first index from favorite contacts i.e. Alex
+image::unfavorite-after-find-command-result.png[width="800"]
+// end::favoriteunfavoriteperson[]
 
 ==== Deleting a person: `delete`
 
-Deletes the specified person(s) from the address book. +
-Command format: `delete [OPTION] KEYWORD [MORE_KEYWORDS]...` +
-Alias: `d`
+To delete person(s) from the address book, use the `delete` command.
+
```

Comment:
I think this format is clearer instead of listing `OPTIONS` separately:

====
Command format:

* Finding by `name`:
`[red]#find# [green]#KEYWORD [ADDITIONAL_KEYWORDS]...#`
* Finding by `tag` [Since v1.3]:
`[red]#find# [blue]#-tag# [green]#KEYWORD [ADDITIONAL_KEYWORDS]...#`
* Finding `favorite` contacts [Coming in v2.0]:
`[red]#find# [blue]#-fav#`

Alias: `[red]#f#` +
====

Reply:
Updated ✅